### ISSUE TITLE
shadowing CSS

### ISSUE THREAD

**Issue by lenowac** [hikerpig/gatsby-project-kb#35]
hey, i'm having a tough time shadowing the theme.css and vars.css.

none of those directory variants work. The site builds, but the CSS is untouched: 
```
src\foam-template-gatsby-kb\styles\*.css
src\foam-gatsby-kb\styles\*.css
src\gatsby-project-kb\styles\*.css
src\gatsby-theme-kb\styles\*.css
gatsby-theme-kb\src\styles\*.css
src\@hikerpig\foam-template-gatsby-kb\styles\*.css
src\@hikerpig\foam-gatsby-kb\styles\*.css
src\@hikerpig\gatsby-project-kb\styles\*.css
```
The source and version history can be found here: https://github.com/lenowac/wiki

I understand that OSS comes without support, but in case you happen to have some spare time, I'd really appreciate a little help. In case you don't, that's very understandable :)

Best regards, lenowac ^^

**Comment by lenowac** [comment]
just noticed that the most obvious way, importing them in gatsby-browser, seems to work, but feels like a hacky workaround: some areas are'nt styled properly

**Comment by hikerpig** [comment]
Thanks for using this theme.
I don't think Gatasby's shadowing feature includes CSS files. And it's right to import your own css file in gatsby-browser to override styles in a plain old _Cascading_ way. I should add this to the instructions.
What do you mean by _except for the graph module_ ? Can you post an example of what you tried to accomplish? Maybe it's my CSS structure that needs to change.

**Comment by hikerpig** [comment]
@lenowac I checked the Gatsby shadowing tutorial and found that it's possible to override the css files so I'm mistaken in the last comment. But in this theme's case, I don't think it's a good idea to shadowing the whole css. We can simply add new  styles to override them.
I've added a new post describing some useful approaches to [override the styles](https://gatsby-project-kb.vercel.app/examples/override-styles), hope it may solve your issue.
